mm2src/common/log.rs

```diff
@@ -971,7 +957,7 @@ impl LogState {
     /// Start intercepting the `log!` invocations happening on the current thread.
     #[cfg(not(target_arch = "wasm32"))]
     pub fn register_my_thread(&self) -> Result<(), String> {
-        let gravity = try_s!(self.gravity.spinlock(77));
+        let gravity = self.gravity.lock();
```

Comment:
`if let Some(ref gravity) = *self.gravity.lock() {`

Reply:
As we discussed before, it's better to leave it as it is to make it more obvious that the `MutexGuard` still exists.